test/it/client-create-application.js

```diff
@@ -49,7 +49,7 @@ describe('client.createApplication()', () => {
   it('should allow me to create a basic authentication application', async () => {
     const application = {
       name: 'template_basic_auth',
-      label: `Sample Basic Auth App - ${faker.random.words()}`,
+      label: `Sample Basic Auth App - ${faker.random.word()}`,
```

Comment:
what's the length of `faker.random.word()`?

Reply:
Limited to the longest word in english dictionary, which after googling turns out to be `Pneumonoultramicroscopicsilicovolcanoconiosis` (45 letters) 😄